### ISSUE TITLE
is the Lyman-alpha forest OK in simqso?

### ISSUE THREAD

**Issue by moustakas** [desihub/desisim#300]
With #293 now merged, @andreufont (and/or other Lya experts, @profxj, @julienguy), could you please take a look at the implementation of the "quick" (i.e., 1D) Lyman-alpha forest implemented in `simqso` and assess whether it's sufficient for DESI simulations?  Note that unlike the `desisim.templates.QSO` class this does *not* use `desisim.lya_mock_p1d.MockMaker`.  

@imcgreer can provide details of the implementation but for reference you can take a look at [simqso.hiforest](https://github.com/imcgreer/simqso/blob/master/simqso/hiforest.py).

Also, in case it's useful, take a look at cells [11], [12], and [17] of [this notebook](https://github.com/desihub/desisim/blob/master/doc/nb/simqso-templates.ipynb).

**Comment by andreufont** [comment]
I will not be able to work on this until Thursday, but we can start the discussion: why is the forest added in simqso? I thought that simqso would just generate unabsorbed quasar continua, and the absorption would be later on outside simqso. 
It is ok if there is the option to add forest, but from the point of view of DESI mocks we would like to set that option to False and take control of how we add the forest (and other absorption fields) somewhere else.

**Comment by moustakas** [comment]
It's because this was an existing, fast, self-consistent, and well-tested capability already in `simqso`. It can of course be turned off trivially with `noforest=True` (as it will be, when using the 3D mocks), but those are pretty compelling reasons for me.

**Comment by andreufont** [comment]
It is good to know that we have this other option to add fluctuations, and we should try to see the effect on things like redshift fits when we use this option instead of others. 
The other fast way to generate 1D mocks (from desisim.lya_mock_p1d.MockMaker) has the advantage that it has a well calibrated 1D power spectrum (it is the code used by Pat McDonald in his 2006 paper), but it does not have extra features like higher order Lyman lines or similar that simqso might have.

**Comment by andreufont** [comment]
I took a look at the code, and it looks like the HI absorbers in simqso are randomly distributed, what means that their power spectrum is pure white noise, what I guess it is fine for most simple tests we want. On the other hand, the absorbers are more realistic here, with actual Voigt profiles, while lya_mock_p1d does not have real absorber profiles.
But as I said above, it would be great if we made it quite clear that the default flow of the code was to ask simqso for the continua only, and apply a HI (and/or others) transmission outside.
